src/codeu/chat/common/ConversationHeader.java

```diff
@@ -48,19 +49,38 @@ public ConversationHeader read(InputStream in) throws IOException {
       );
 
     }
+    //assigning permission levels to users 
   };
 
+  public Integer permissionLevel(String title) throws IOException{
+    if (title.equals("member")){
+      return 1;
+    }
+    else if (title.equals("owner")){
+      return 2;
+    }
+    else if (title.equals("creator")){
+      return 3;
+    }
+    else {
+      throw new IOException("Please enter a valid user category.");
```

Comment:
Not sure if IOException is the right kind of exception to use here. May be IllegalArgumentException ?